src/zoid/card-fields/component.jsx

```diff
@@ -48,11 +48,11 @@ const CARD_FIELD_TYPE = {
 };
 
 type InstallmentsConfiguration = {|
-  financingCountryCode : string,
-  currencyCode : string,
-  billingCountryCode : string,
-  amount : string,
-  includeBuyerInstallments ? : boolean
+  financingCountryCode: string,
+  currencyCode: string,
+  billingCountryCode: string,
+  amount: string,
+  includeBuyerInstallments?: boolean,
```

Comment:
nit: do we want to remove the lint updates out of this PR for clarity?

Reply:
Good call. I can create a separate PR for the lint commits.

Reply:
We can leave it in for this one. But also the comment, so it is easier to trace back that it is simply a lint update and not actual code changes.